VAMobile/src/components/Templates/HeaderBanner.test.tsx

```diff
@@ -40,74 +33,54 @@ context('HeaderBanner', () => {
       }
     }
 
-    component = render(<HeaderBanner leftButton={leftButton} title={title} rightButton={rightButton} />)
-
-    testInstance = component.UNSAFE_root
+    render(<HeaderBanner leftButton={leftButton} title={title} rightButton={rightButton} />)
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
   describe('title', () => {
-    it('should not be there', async () => {
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews.length).toEqual(0)
-    })
-    it('should be there when title added', async () => {
+    it('should be there when title added', () => {
       initializeTestInstance('test')
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews.length).toEqual(1)
-      expect(textViews[0].props.children).toEqual('test')
+      // TODO: why does getByText fail? 
+      // expect(screen.getByText(/test/)).toBeTruthy()
+      expect(screen.UNSAFE_getByType(TextView).props.children).toEqual('test')
+      expect(screen.getByRole('header', { name: 'test' })).toBeTruthy()
     })
   })
   describe('left banner button', () => {
-    it('should not be there', async () => {
-      const leftButton = testInstance.findAllByType(TouchableWithoutFeedback)
-      expect(leftButton.length).toEqual(0)
-    })
-    it('should be there when text and on press is added', async () => {
+    it('should be there when text and on press is added', () => {
       initializeTestInstance(undefined, 'cancel', onPressSpy)
-      const leftButton = testInstance.findAllByType(TouchableWithoutFeedback)
-      expect(leftButton.length).toEqual(1)
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews.length).toEqual(1)
-      expect(textViews[0].props.children).toEqual('cancel')
+      expect(screen.getByRole('button', { name: 'cancel' })).toBeTruthy()
```

Comment:
technically this test is duplicated in the fireEvent line as it won't work if it isn't there. not necessary to remove this one but you can

Reply:
Left it in, because why not. :D